docs/pages/reference/troubleshooting/installation.md

```diff
@@ -38,3 +38,26 @@ For details about the state of the Meshery Server subscription see the http moni
 #### See Also
 
 - [Error Code Reference](/reference/error-codes)
+
+## Meshery Remote Providers
+
+Once Meshery is installed, the remote provider "Meshery" can be chosen from UI or by using command `mesheryctl system login`:
+
+![Providers](/assets/img/providers/provider_screenshot.png)
```

Comment:
I think there seems to be an issue with the image. It isn't visible to me in the preview mode.